### ISSUE TITLE
please add "extra info from pdf" function

### ISSUE THREAD

**Issue by nerdDan** [pubs/pubs#101]
repo [1] can extract meta info (doi, arxiv) from pdf. please consider integrate it into pubs.

[1] https://github.com/metachris/pdfx

**Comment by omangin** [comment]
Thanks for the suggestion @nerdDan. Can you be more explicit about what you'd like `pdfx` to be used for within pubs? Do you mean using it for guessing the title and authors at import time or something to do about the references contained in the pdf?

**Comment by nerdDan** [comment]
Yes, guessing metadata at import time is exactly what I mean.
This feature is included by many reference managers such as Mendeley and Zotero.

It would be great if you can import a file with this command:
```
pubs add -d article.pdf
```

**Comment by omangin** [comment]
Yes, that would indeed be a nice feature to have. I am not sure yet about which back end is the best suited for this. [Pdfx](https://github.com/metachris/pdfx) is one option, but `pdfinfo`  and `exiftool` also do the job and have more chances of being already installed.

I'll have a look at these, please fill free to comment suggestions or a pull request…

**Comment by beuerle** [comment]
If you mainly import papers/references, where the DOI is given as

```
Digital Object Identifier 00.0000/IEEE.2018.0000000
```

(as is the case for most IEEE papers), you can use this hacky script `pdfimport.sh`:

```bash
#!/usr/bin/zsh

DOI=$(pdftotext $1 - \
	| grep 'Digital Object Identifier' \
	| awk '{ print $4 }')
echo $DOI
pubs add -D $DOI -d $1
```

and then define an alias in `pubsrc` (adjust the path(s) accordingly):

```config
[[alias]]
pdfimport = !~/.pubs/.scripts/pdfimport.sh $1
```

This is of course extendable.
